lib/api.js

```diff
@@ -232,6 +232,13 @@ class Api extends Emittery {
 				}
 
 				const worker = fork(file, options, apiOptions.nodeArguments);
+				worker.onStateChange(data => {
+					if (data.Timeout && !apiOptions.debug)
```

Comment:
The pattern is to identify the state changes using a `type` property. Perhaps `'test-timeout-configured'`? Then pass the timeout along in a separate property. `period` would make it match the existing `timeout` event.

Reply:
Updated. Thanks for pointing out.